lib/msf/core/payload/python/meterpreter_loader.rb

```diff
@@ -264,28 +262,46 @@ def _s2b(text):
 		return[ord(c)for c in text]
 	def _b2s(binary):
 		return bytes(binary)
-	def _concat_list(a,b):
-		return a+bytes(b)
 else:
 	bytes=lambda s,e:s
+def _gmul(a, b):
+	r=0
+	while b:
+		if b&1: r^=a
+		a<<=1
+		if a>255: a^=0x11B
+		b>>=1
+	return r
+def _mix(n, vec):
+	return sum([_gmul(n,v)<<(24-8*shift) for shift,v in enumerate(vec)])
```

Comment:
`return sum(_gmul(n,v)<<(24-8*shift) for shift,v in enumerate(vec))` can be used

Reply:
Done.  The list comprehension has been changed to a generator.